internal/flink/command_artifact_create.go

```diff
@@ -30,14 +30,14 @@ func (c *command) newCreateCommand() *cobra.Command {
 		RunE:  c.createArtifact,
 		Example: examples.BuildExampleString(
 			examples.Example{
-				Text: `Create Flink UDF artifact "my-flink-artifact".`,
+				Text: `Create Flink artifact "my-flink-artifact".`,
 				Code: "confluent flink artifact create my-flink-artifact --artifact-file /Users/xyz/Documents/config/plugin.jar",
```

Comment:
I think most people would use a relative path here, so can this example default to that?